src/org/traccar/api/resource/AttributeResource.java

```diff
@@ -85,15 +85,11 @@
 
     }
 
-    private Response add(Attribute entity) throws SQLException {
+    @POST
+    @Path("test")
```

Comment:
Had to move test functionality in sub-path (actually it is more consistent with others) because we can't remove parent `@POST` annotation in child and can't override `add` with additional parameter.